sio-digital-team/active-safe
# Updated prevent section tabs to be mobile responsive and still functi…

https://trello.com/c/6zrqspl6

**Changes:**

- [x] Updated template file for mobile first approach and moved tab content to be within its parent toggle 
- [x] Updated cssstyling to adjust for change in the template
- [x] Updated JS to reflect change and reworked the toggling and screen resizing
 
**How to test:**
- Ensure there is content in the tabs to test and view on **Desktop**
- Scroll to the prevent section and toggle to tabs to make sure they're opening and closing correctly
- Resize the browser to **mobile** and the currently active tab is still opened and you are now able to open and close the accordions
- Resize back to desktop and test if tabs still open correctly onload
